src/main/java/hsien/Hsien.java

```diff
@@ -0,0 +1,260 @@
+package hsien;
+
+import hsien.task.*;
+import hsien.exception.*;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.util.regex.*;
+
+public class Hsien {
+
+    public static void printLogo() {
+        System.out.println(" _   _         _____        _ _   _ ");
+        System.out.println("| | | | ##### |_   _|##### |   \\ | |");
+        System.out.println("| |_| |#        | |  #     | |\\ \\| |");
+        System.out.println("|  _  | #####   | |  ##### | | \\ \\ |");
+        System.out.println("| | | |      # _| |_ #     | |  \\  |");
+        System.out.println("|_| |_| ##### |_____|##### |_|   \\_|\n");
+    }
+
+    public static void printLine() {
+        System.out.println("\n" + "-".repeat(50) + "\n");
+    }
+
+    public static void welcomeMessage() {
+        printLine();
+        printLogo();
+        // Greet
+        System.out.println("Hello! I am Hsien, your personal chatbot...");
+        printLine();
+    }
+
+    public static void printList(ArrayList<Task> tasks) {
+        if (tasks.isEmpty()) {
+            System.out.println("List is currently empty. Please add a task!");
+            return;
+        }
+        int counter = 1;
+        System.out.println("Here are the tasks in your list!");
+        for (Task t : tasks) {
+            System.out.printf("%s. %s%n", counter, t.getStatusDescription());
+            counter += 1;
+        }
+    }
+
+    public static void printCommands(List<String> validCommands) {
+        System.out.println("These are the possible commands:");
+        for (int i=1; i<= validCommands.size(); i+=1) {
+            System.out.printf("%d. %s\n", i, validCommands.get(i - 1));
+        }
+        System.out.print("Please enter a command/add task (type 'bye' to exit): ");
```

Comment:
Consider making the print statement sentences into strings. This will reduce any typo and makes the line reusable, modular and easy to manage.